.github/workflows/docker-image.yml

```
@@ -0,0 +1,18 @@
+    branches: [ "main" ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Build the Docker image
+      run: docker build . --file Dockerfile --tag my-image-name:$(date +%s)
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:1:54: Quote this to prevent word splitting

(shellcheck)